src/PageContent/FoodLogs/FoodLogsContainer.js

```diff
@@ -1,193 +1,114 @@
-                                unit: 'lb'
-                            },
-                            {
-                                food: 'apple',
-                                quantity: '50',
-                                unit: 'lb'
-                            },
-                            {
-                                food: 'orange',
-                                quantity: '50',
-                                unit: 'lb'
-                            },
-                            {
-                                food: 'pineapple',
-                                quantity: '50',
-                                unit: 'lb'
-                            },
-                            {
-                                food: 'carrot',
-                                quantity: '2',
-                                unit: 'lb'
-                            },
-                            {
-                                food: 'watermelon',
-                                quantity: '32',
-                                unit: 'cases'
-                            },
-                            {
-                                food: 'cupcakes',
-                                quantity: '200',
-                                unit: 'lb'
-                            },
-                            {
-                                food: 'grapes',
-                                quantity: '2',
-                                unit: 'loaves'
-                            }
-                        ]
-                    },
-                    notes: '4/9, mon x4, 1-3p, none requested',
-                    raContact: {
-                        email: 'refigerator@uniongospel.org',
-                        name: 'Bob Vance',
-                        phone: '111-222-3333',
-                    },
-                    receivingAgency: { //uid replaced by info
-                        name: 'Seattle Union Gospel'
-                    },
-                    pickedUpInfo: {
-                        temperature: 29,  // in F
-                        signature: 'John Smith',
-                        timestamp: 1523174874685
-                    },
-                    deliveredInfo: {
-                        signature: 'Ellen Blake',
-                        timestamp: 1523174892769
-                    }
-                }
-            ]
+
+    async componentDidMount(){
+        const rawDeliveries = await this.fetchDeliveries();
+        this.aggrFoodLogsInfo(rawDeliveries);
+    }
+
+    async fetchDeliveries() {
+        const { account } = this.props; 
+        const agencyUid = 
+            account.accountType === AccountType.DONATING_AGENCY_MEMBER ? account.agency : account.uid; 
+        // get all deliveries' id under this agency 
+        const deliveriesIdList = [];
+        const deliveriesIdSnapshot = await deliveryIndicesRef.child(`${account.umbrella}/${agencyUid}`).once('value');
```

Comment:
Here you could only query for the deliveries in the past by adding `.orderByKey().endAt(now)`